-------------------------------------------------------------------------------
Test set: mx.tec.lab.UnderstandingAssertsApplicationTests
-------------------------------------------------------------------------------
Tests run: 10, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.615 s <<< FAILURE! - in mx.tec.lab.UnderstandingAssertsApplicationTests
whenCheckingExceptionMessage_thenEqual  Time elapsed: 0.009 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Exception not thrown
	at mx.tec.lab.UnderstandingAssertsApplicationTests.whenCheckingExceptionMessage_thenEqual(UnderstandingAssertsApplicationTests.java:87)

